Classes/IndexingJob.php

```diff
@@ -30,7 +30,7 @@ public function execute(QueueInterface $queue, Message $message)
 
                 // Skip this iteration if the nodedata can not be fetched (deleted node)
                 if (!$nodeData instanceof NodeData) {
-                    $this->log(sprintf('action=indexing step=failed node=%s message="Node data could not be loaded"', $node['identifier']), \LOG_ERR);
+                    $this->log(sprintf('action=indexing step=failed node=%s message="Node data could not be loaded"', $node['identifier']), \LOG_WARNING);
```

Comment:
Maybe this logging call could even be removed completely? Or made `LOG_INFO`?

Reply:
If I got it right, we have a node in a batch which was deleted in the meantime. That could happen and is not a real error - I'm fine with both.

Reply:
Ok, let's keep it as a `NOTICE`